Components/Spinner/index.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from "prop-types";
+import React from "react";
+import styles from "./styles.css";
+
+const Spinner = ({ enabled, title }) => {
+  if (enabled) {
```

Comment:
Maybe this prop should be called `visible`? For example, a button can be disabled but still be visible. This would keep the behaviour consistent with other components.

Reply:
That sounds pretty reasonable.

Reply:
Changed.